service/dap/server.go

```diff
@@ -732,6 +748,7 @@ func (s *Server) onConfigurationDoneRequest(request *dap.ConfigurationDoneReques
 	}
 	s.send(&dap.ConfigurationDoneResponse{Response: *newResponse(request.Request)})
 	if !s.args.stopOnEntry {
+		//TODO: it's probably a good idea to ignore this on a multiclient server
```

Comment:
Why? How is this different from a user clicking continue manually? This is just a shortcut. (Also this is the default behavior in vscode)

Reply:
If the program is already running this will wait until it stops and the restart it